protected/controllers/PatientController.php

```diff
@@ -1507,8 +1507,8 @@ public function actionCreate()
             $patient->attributes = $_POST['Patient'];
             $address->attributes = $_POST['Address'];
 
+            $referral = new PatientReferral();
```

Comment:
It's part of the actionCreate() function. From my testing it doesn't seem to matter